.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Mulling this over, I think any push to main is valid.  That being said, we could have it trigger on tags, and for now only allow that in the main branch, then in the future figure out rules for other branches and make this more permissive.

Reply:
re: triggering on tags, can you describe that in more detail in [this Confluence page](https://midetech.atlassian.net/wiki/spaces/EC/pages/1410105359/How+should+the+deployment+action+trigger)? I tried to do it justice but I'm biased against the idea tbh.